spec/mongoid/association/referenced/has_many/enumerable_spec.rb

```diff
@@ -201,6 +201,122 @@
     end
   end
 
+  describe "#empty?" do
+
+    let(:person) do
+      Person.create
+    end
+
+    let!(:post_one) do
+      Post.create(person_id: person.id)
+    end
+
+    let!(:post_two) do
+      Post.create(person_id: person.id)
+    end
+
+    context "when only a criteria target exists" do
+
+      let(:criteria) do
+        Post.where(person_id: person.id)
+      end
+
+      let!(:enumerable) do
+        described_class.new(criteria)
+      end
+
+      let!(:empty) do
```

Comment:
Why does this use `let!` instead of `let`?

Reply:
Because I copied the spec from `#any` and it was done that way there. I've changed it for both.